linkis-computation-governance/linkis-manager/linkis-manager-persistence/src/main/java/org/apache/linkis/manager/persistence/impl/DefaultNodeManagerPersistence.java

```diff
@@ -351,19 +351,22 @@ public List<EngineNode> getEngineNodeByServiceInstance(List<ServiceInstance> ser
 
   @Override
   public List<Node> getNodesByOwnerList(List<String> ownerlist) {
-    List<PersistenceNode> nodeInstances = nodeManagerMapper.getNodeInstancesByOwnerList(ownerlist);
     List<Node> persistenceNodeEntitys = new ArrayList<>();
-    if (!nodeInstances.isEmpty()) {
-      for (PersistenceNode persistenceNode : nodeInstances) {
-        PersistenceNodeEntity persistenceNodeEntity = new PersistenceNodeEntity();
-        ServiceInstance serviceInstance = new ServiceInstance();
-        serviceInstance.setApplicationName(persistenceNode.getName());
-        serviceInstance.setInstance(persistenceNode.getInstance());
-        persistenceNodeEntity.setServiceInstance(serviceInstance);
-        persistenceNodeEntity.setMark(persistenceNode.getMark());
-        persistenceNodeEntity.setOwner(persistenceNode.getOwner());
-        persistenceNodeEntity.setStartTime(persistenceNode.getCreateTime());
-        persistenceNodeEntitys.add(persistenceNodeEntity);
+    if (CollectionUtils.isNotEmpty(ownerlist)) {
+      List<PersistenceNode> nodeInstances =
+          nodeManagerMapper.getNodeInstancesByOwnerList(ownerlist);
+      if (CollectionUtils.isNotEmpty(nodeInstances)) {
+        for (PersistenceNode persistenceNode : nodeInstances) {
+          PersistenceNodeEntity persistenceNodeEntity = new PersistenceNodeEntity();
+          ServiceInstance serviceInstance = new ServiceInstance();
+          serviceInstance.setApplicationName(persistenceNode.getName());
+          serviceInstance.setInstance(persistenceNode.getInstance());
+          persistenceNodeEntity.setServiceInstance(serviceInstance);
+          persistenceNodeEntity.setMark(persistenceNode.getMark());
+          persistenceNodeEntity.setOwner(persistenceNode.getOwner());
+          persistenceNodeEntity.setStartTime(persistenceNode.getCreateTime());
+          persistenceNodeEntitys.add(persistenceNodeEntity);
+        }
       }
     }
     return persistenceNodeEntitys;
```

Comment:
add log info for controller api  
org.apache.linkis.manager.rm.restful.RMMonitorRest#getQueueResource 

![image](https://github.com/WeDataSphere/linkis/assets/7869972/b141e09c-a84e-423e-bb93-c8c48f5589f4)

Reply:
add audit log for api   /linkisManager/rm/queueresources

Reply:
done